birdhouse/config/jupyterhub/jupyterhub_config.py.template

```diff
@@ -13,7 +13,7 @@ c.MagpieAuthenticator.magpie_url = "http://magpie:2001"
 c.JupyterHub.cookie_secret_file = '/persist/jupyterhub_cookie_secret'
 c.JupyterHub.db_url = '/persist/jupyterhub.sqlite'
 
-c.JupyterHub.template_paths = ['/template_paths']
+c.JupyterHub.template_paths = ['custom_templates']
```

Comment:
Lost the `/` in front?  Do you really need to rename?

Reply:
Yes, good catch. I thought template_paths was the name of the configuration parameter, containing a list of paths. it made more sense to me to call it like a folder that contains templates, but it's not required...

Reply:
It's okay, keep the rename.